src/app/shared/components/icon-button/icon-button.component.ts

```diff
@@ -22,6 +22,9 @@ export class IconButtonComponent implements OnInit, OnChanges {
   @HostBinding('attr.disabled')
   public disabled: boolean = false
 
+  @Input()
+  public disabledTooltip: string = ''
```

Comment:
```suggestion
  @Input() public disabledTooltip: string = ''
```

Just a suggestion :)